egs/wsj/s5/steps/libs/nnet3/xconfig/basic_layers.py

```diff
@@ -1075,6 +1087,59 @@ def get_full_config(self):
         return ans
 
 
+class XconfigExistingLayer(XconfigLayerBase):
+    """
+    This class is for lines like 
+    'existing name=tdnn1.affine dim=40'.
```

Comment:
This documentation is not meaningful.  Where I wrote:
```
 This class is for lines like 
     xxxxxx
```
the line xxxxx would  appear in an xconfig file.  But these wouldn't appear in an xconfig file.
I don't want to have to give you the documentation word for word.  Figure it out.